README.md

```diff
@@ -16,18 +16,29 @@ downstream repos.
 
 ## Configuration
 
-1. Create a `robo.yml` file in the root of your codebase. `robo.drupal.example.yml`
+- Create a `robo.yml` file in the root of your codebase. `robo.drupal.example.yml`
 is provided as a starting point for Drupal projects.
-1. Create a `.sites.config.yml` file in the root of your codebase. See
+- Create a `.sites.config.yml` file in the root of your codebase. See
 `.sites.config.example.yml` for reference on what can/should be configured.
-1. Add the following to your repo's `composer.json` "scripts" section so that you
-can call robo easily with `composer robo`:
-
-```json
-"scripts": {
+- Add the following to your repo's `composer.json` "scripts" section so that you can call robo easily with `composer robo`:
+  ```json
+  "scripts": {
     "robo": "robo --ansi"
-}
-```
+  }
+  ```
+- If you are overriding any commands, add the following to your repo's `composer.json` file.
+  ```json
+  "autoload": {
+  "psr-4": {
+      "BPRobo\\": "robo/src"
```

Comment:
This and the reference to BPRobo below should be removed. This is going to depend on whatever namespace the consumer is setting.

Reply:
Ah, yes, I missed that initial in there!

Should it be `ExampleRobo` or something generic then?

Reply:
```suggestion
      "YOURPREFIXRobo\\": "robo/src"
```